C-Plus-Plus/dp/Maximum_area_Square_with_all_1.cpp

```diff
@@ -0,0 +1,59 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
```

Comment:
Try to modularize the Code here. Push all the functional code in a separate function and use main() for the Driver Code.